theme_notes/views/snippets/s_carousel_intro.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_carousel_intro" inherit_id="website.s_carousel_intro">
+    <!-- Layout -->
+    <xpath expr="//div[hasclass('carousel')]" position="attributes">
+        <attribute name="class" remove="carousel-dark" separator=" "/>
+        <attribute name="data-oe-shape-data">{'shape':'web_editor/Airy/02','flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}</attribute>
+    </xpath>
+    <xpath expr="//div[hasclass('carousel-inner')]" position="before">
+        <div class="o_we_shape o_web_editor_Airy_02"/>
+    </xpath>
```

Comment:
I don't see the shape here.
![image](https://github.com/user-attachments/assets/e39d9b58-ddb9-4844-833d-84901c3d93b3)
Probably because the slide is on top of the carousel container. If you put the shape on the slide, it works, but only for this slide.

Reply:
Good catch ! I removed the shape 👍